api/v1/aerospikecluster_validating_webhook.go

```diff
@@ -119,7 +119,7 @@ func (c *AerospikeCluster) ValidateUpdate(oldObj runtime.Object) (admission.Warn
 	if err := validateAerospikeConfigUpdate(
 		aslog, incomingVersion, outgoingVersion,
 		c.Spec.AerospikeConfig, old.Spec.AerospikeConfig,
-		c.Status.AerospikeConfig,
+		c.Status.AerospikeConfig, c.Status.Pods,
```

Comment:
This function is being called from `validateRackUpdate` where instead of passing `c.Status.AerospikeConfig`, we pass `rackStatusConfig`. So here we can pass the whole status object but in rack-specific validation, it will be difficult.